pkgs/test/test/runner/retry_test.dart

```diff
@@ -2,8 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'package:test/test.dart';
 @TestOn('vm')
+
```

Comment:
You *could* add a `library _;` here. Just to show good sportsmanship :)

Reply:
Might be a good idea, but we should probably do the whole repo if we do it.